<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Utama</string>
  </property>
  <widget class="QPushButton" name="btnMesjid">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Mesjid</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAcara">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Acara </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPengunjung">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Pengunjung</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKamera">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Kamera</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM UTAMA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
